src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/store/application.ts

```diff
@@ -50,6 +50,9 @@ export const useApplicationStore = defineStore("application", {
         }, 0) ?? 0
       );
     },
+    draftApplicationIncludesCertification: (state) => {
```

Comment:
What you have works and OK to leave as is, but we don't really need to pass the certificate type into the getter we should have everything we need already in the store. What I had in mind was:

`isIte: (state) => {}`
`isSne: (state) => {}`
`isFiveYear: (state) => {}`
`IsOneYear: (state) => {}`
`isAssistant: (state) => {}`

This would make the logic checks a bit easier to digest:
`if (this.applicationStore.draftApplicationIncludesCertification(CertificationType.FIVE_YEAR))`
becomes
`if (this.applicationStore.isFiveYear)`

Reply:
Or maybe putting draft in there is a good idea i.e. `draftIsIte`, etc...

Reply:
Yeah I think adding draft in there would be good. Since we might have to start referring to application sometimes too.